mne/io/fieldtrip.py

```diff
@@ -0,0 +1,345 @@
+    This field can have one or more columns. The function first removes
+    columns according to the two omit parameters.
+
+    - If only one column remains, its values are used as event values in the
+      MNE Epoch.
+    - If two or more columns remain, each unique combination of
+      these values receives a new event value. These event values are created
+      automatically. In order to match these to conditions, you can use the
+      trialinfo_map parameter.
+
+    Parameters
+    ----------
+    ft_structure_path: str
+        Path and filename of the .mat file containing the data.
+    data_name: str
+        Name of heading dict/ variable name under which the data was originally
+        saved in MATLAB.
+    trialinfo_map: dict
+        A dictionary mapping condition strings (MNE's event_ids) to the
+        trialinfo column. The values should be 1D numpy arrays. See examples
+        for details.
+    omit_trialinfo_index: bool
+        Omit trialinfo columns that look like an index of the trials, i.e. in
+        which every row is the row before + 1.
+    omit_non_unique_trialinfo_index: bool
+        Omit trialinfo columns that contain a different value for each row. T
+        ese are most likely additional data like reaction times that cannot
+        be represented in MNE.
+
+
+    Returns
+    -------
+    mne.EpochsArray
+        A MNE EpochsArray structure consisting of the epochs arrays, an event
+        matrix, start time before event (if possible, else defaults to 0) and
+        measurement info.
+
+    Examples
+    -------
+    >>> read_epochs_fieldtrip('FieldTripEpochsFile', # doctest: +SKIP
+    >>>     trialinfo_map={ # doctest: +SKIP
+    >>>         'audio/attend': np.array([0, 1]), # doctest: +SKIP
+    >>>         'visual/attend': np.array([1, 1]), # doctest: +SKIP
+    >>>         'audio/non_attend': np.array([0, 0]), # doctest: +SKIP
+    >>>         'visual/non_attend': np.array([1, 0])}) # doctest: +SKIP
+    """
+    pymatreader = _check_pymatreader()
+
+    ft_struct = pymatreader.read_mat(ft_structure_path,
+                                     ignore_fields=['previous'],
+                                     variable_names=[data_name])
+
+    # load data and set ft_struct to the heading dictionary
+    ft_struct = ft_struct[data_name]
+
+    data = np.array(ft_struct['trial'])  # create the epochs data array
+    (events, event_id) = _create_events(ft_struct, trialinfo_map,
+                                        omit_trialinfo_index,
+                                        omit_non_unique_trialinfo_index)
+    tmin = _set_tmin(ft_struct)  # create start time
+    info = _create_info(ft_struct)  # create info structure
+
+    custom_epochs = mne.EpochsArray(data=data, info=info, tmin=tmin,
+                                    events=events, event_id=event_id)
+    return custom_epochs
+
+
+def read_evoked_fieldtrip(ft_structure_path, comment='', data_name='data'):
```

Comment:
I would make the default `comment=None`, meaning "use the `op.basename` of the path" (assuming there is no better default already in the FT .mat file)